resources/visualizations/cases/cases.model.js

```diff
@@ -11,4 +11,14 @@ const queryMapData = () =>
     )
     .then((queryResult) => queryResult.rows);
 
-module.exports = { queryMapData };
+const queryByCountryData = () =>
+  db
+    .raw(
+      "SELECT * " +
+        'FROM "covidall" ' +
+        "WHERE EXISTS (SELECT confirmed WHERE confirmed >= 90000) " +
+        'AND date >= "2020-04-28 00:00:00" '
+    )
+    .then((queryResult) => queryResult.rows);
```

Comment:
If you didn't want to use a raw SQL query, I think you could do:
`db("covidall").where("confirmed", ">=", 90000).andWhere("date", ">=", "2020-04-28 00:00:00")`

I had to use a raw statement in queryMapData because knex didn't have a built in method for casting bigint to int. In this case, I think you might be able to chain knex methods and avoid a big SQL string. Either way works perfectly fine though.